azimuth_capi/operator.py

```diff
@@ -417,6 +416,11 @@ async def on_cluster_create(logger, instance, name, namespace, patch, **kwargs):
     if instance.spec.paused:
         logger.info("reconciliation is paused - no action taken")
         return
+
+    # To help trigger timeouts if we get stuck, save the last_updated_timestamp
+    if not instance.status.last_updated_timestamp:
+        await save_cluster_status(instance)
+
```

Comment:
This should _always_ reset to the current timestamp, because an update has just been applied.

Reply:
That stops us detecting an error loop where we are stuck retrying the update over and over again.